services/api-server/src/simcore_service_api_server/core/application.py

```diff
@@ -5,10 +5,16 @@
 from models_library.basic_types import BootModeEnum
 from packaging.version import Version
 from servicelib.fastapi.profiler_middleware import ProfilerMiddleware
-from servicelib.fastapi.tracing import setup_tracing
+from servicelib.fastapi.tracing import (
+    setup_tracing,
+    setup_tracing_spans_for_simcore_service_functions,
+)
 from servicelib.logging_utils import config_all_loggers
+from simcore_service_api_server import exceptions
```

Comment:
we use relative imports ... `` from .. import exceptions``
Actually, perhaps you can do some cleanup? :-)
![image](https://github.com/user-attachments/assets/8977be2f-cd8f-4382-868f-dac8b805b8ef)

Exceptionally  `main` might need absolute import, otherwise we should use relavite imports everywhere.